README.md

```diff
@@ -1,2 +1,47 @@
-# alf
-Agent Learning Framework
+# ALF
+Agent Learning Framework (ALF) is a reinforcement learning framework
+emphasizing on the easeness of writing complex model architectures. ALF is built
+on [Tensorflow 2.0](https://www.tensorflow.org/beta/)
+
+## Features
+
+
+## Algorithms
+
+
+
+* [A2C](alf/algorithms/actor_critic_algorithm.py): [OpenAI Baselines:
+ACKTR & A2C](https://openai.com/blog/baselines-acktr-a2c/)
+* [DDPG](alf/algorithms/ddpg_algorithm.py): Lillicrap et al. "Continuous control with deep reinforcement learning" [arXiv:1509.02971](https://arxiv.org/abs/1509.02971)
+* [PPO](alf/algorithms/ppo_algorithm.py): Schulman et al. "Proximal Policy Optimization Algorithms" [arXiv:1707.06347](https://arxiv.org/abs/1707.06347)
+* [SAC](alf/algorithms/sac_algorithm.py): Haarnoja et al. "Soft Actor-Critic Algorithms and Applications" [arXiv:1812.05905](https://arxiv.org/abs/1812.05905)
+* [ICM](alf/algorithms/icm_algorithm.py): Pathak et al. "Curiosity-driven Exploration by Self-supervised Prediction" [arXiv:1705.05363](https://arxiv.org/abs/1705.05363)
+* [MERLIN](alf/algorithms/merlin_algorithm.py): Wayne et al. "Unsupervised Predictive Memory in a Goal-Directed Agent"[arXiv:1803.10760](https://arxiv.org/abs/1803.10760)
+
+## Installation
+
+You can run the following commands to install ALF
+```
+git clone https://github.com/HorizonRobotics/alf
+cd alf
+git submodule update --init --recursive
+cd tf_agents
+pip install -e .
+cd ..
+pip install -e .
+```
+
+## Examples
+
+### A2C
```

Comment:
Do you think it's necessary that we put game snapshots, reward curves, and statistics for each example?

Reply:
Yes, we should do that. And we should have some video showing how the agents perform.